int64 imgID
sensor_msgs/RegionOfInterest roi
---
sensor_msgs/Image data